### ISSUE TITLE
In a live acquisition the first imaged autoROI plane is larger than the preview 

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/BakingTray#253]
Set up sample with autoROI. The sample is found correctly but the the first plane ends up being imaged far too large. It actually seems to add tiles. After that the the autoROI behaves as expected.

**Comment by raacampbell** [comment]
This happened on the first two acquisitions I tried: `RC_LR_01_autoTest` and the 5 section acq before that. It didn't happen in  `RC_LR_02_autoTest`. I don't think I changed the code in between these.

**Comment by raacampbell** [comment]
This happened on `IO_116_AUTOROI_TEST`, where I see:

```
01/05 180 tiles, 12 x 15
02/05 80 tiles, 8 x 10
03/05 80 tiles, 8 x 10
04/05 80 tiles, 8 x 10
05/05 80 tiles, 8 x 10
```

The recipe was set up for 11 by Y 14 tiles.

**Comment by raacampbell** [comment]
Running `populateCurrentTilePattern` instead of `getNextROIs` at the beginning of an acquisition seems to get rid of the problem.

**Comment by raacampbell** [comment]
This commit in the autofinder repo fixes this bug: 813cf64764ac550e4692fd12c81de65dce3a2335
